tests/gtest/avifreadimagetest.cc

```diff
@@ -88,6 +88,138 @@ TEST(PngTest, RgbColorTypeWithTrnsBeforePlte) {
   EXPECT_EQ(image->alphaPlane, nullptr);
 }
 
+constexpr size_t colorProfileSize = 376;
+constexpr size_t grayProfileSize = 275;
```

Comment:
The C++ code in the tests/gtest directory follows the Google C++ Style Guide. So these two constants should be named `kColorProfileSize` and `kGrayProfileSize`.